src/components/list/list.js

```diff
@@ -244,6 +244,7 @@ function mdListItemDirective($mdAria, $mdConstant, $mdUtil, $timeout) {
           wrapIn('div');
         } else if (!tEl[0].querySelector('md-button:not(.md-secondary):not(.md-exclude)')) {
           tEl.addClass('md-no-proxy');
+          tEl.addClass('md-list-item-content');
```

Comment:
Yeah that's intentionally. It allows us to easily apply the custom styles for user content.